app/views/modals/paymentsModal.html

```diff
@@ -436,6 +436,7 @@ <h4 translate>Refund Payment</h4>
           <select ng-model="newTransaction.paymentType" class="form-control" ng-change="setTransactionAmount()">
             <option value="" disabled selected translate>Choose one...</option>
             <optgroup label="Payment">
+              <option value="CREDIT_CARD" ng-if="conference.paymentGatewayKeySaved" translate>Credit Card</option>
```

Comment:
This is probably from a merge commit. This needs to be gone. Lee removed it a few days ago.